### PR TITLE
upd readme for py/classification_demo

### PR DESCRIPTION
**Pull request by ivikhrev**
Added missing arguments, fixed descritption for some. It's a copy-paste from console, so formatting also hooked up.

### CODE REVIEW

File: demos/classification_demo/python/README.md [hunk 1]
```diff
@@ -130,60 +130,82 @@ and `<omz_dir>/data/dataset_classes/imagenet_2012.txt` labels file with all othe
...
 
 Input transform options:
   --reverse_input_channels
-                        Optional. Switch the input channels order from BGR to RGB.
+                        Optional. Switch the input channels order from BGR to
+                        RGB.
   --mean_values MEAN_VALUES MEAN_VALUES MEAN_VALUES
-                        Optional. Normalize input by subtracting the mean values per channel. Example: 255 255 255
+                        Optional. Normalize input by subtracting the mean
+                        values per channel. Example: 255 255 255
```
Comment (anzhella-pankratova) [inline demos/classification_demo/python/README.md:204]:
```suggestion
                        values per channel. Example: 255.0 255.0 255.0
```

File: demos/classification_demo/python/README.md [hunk 2]
```diff
@@ -130,60 +130,82 @@ and `<omz_dir>/data/dataset_classes/imagenet_2012.txt` labels file with all othe
...
   --scale_values SCALE_VALUES SCALE_VALUES SCALE_VALUES
-                        Optional. Divide input by scale values per channel. Division is applied after mean values
-                        subtraction. Example: 255 255 255
+                        Optional. Divide input by scale values per channel.
+                        Division is applied after mean values subtraction.
+                        Example: 255 255 255
```
Comment (anzhella-pankratova) [inline demos/classification_demo/python/README.md:208]:
```suggestion
                        Example: 255.0 255.0 255.0
```

File: demos/classification_demo/python/README.md [hunk 3]
```diff
@@ -130,60 +130,82 @@ and `<omz_dir>/data/dataset_classes/imagenet_2012.txt` labels file with all othe
 Running the application with the `-h` option yields the following usage message:
 
 ```
-usage: classification_demo.py [-h] -m MODEL -i INPUT [-d DEVICE] --labels LABELS [-ntop {1,2,3,4,5,6,7,8,9,10}]
-                              [-nireq NUM_INFER_REQUESTS] [-nstreams NUM_STREAMS] [-nthreads NUM_THREADS] [--loop]
-                              [-o OUTPUT] [-limit OUTPUT_LIMIT] [--no_show] [--output_resolution OUTPUT_RESOLUTION]
-                              [-u UTILIZATION_MONITORS] [--reverse_input_channels]
+usage: classification_demo.py [-h] -m MODEL [--adapter {openvino,remote}] -i
+                              INPUT [-d DEVICE] [--labels LABELS]lues showing.
```
Comment (anzhella-pankratova) [inline demos/classification_demo/python/README.md:134]:
```suggestion
                              INPUT [-d DEVICE] [--labels LABELS]

File: demos/classification_demo/python/README.md [hunk 4]
```diff
@@ -130,60 +130,82 @@ and `<omz_dir>/data/dataset_classes/imagenet_2012.txt` labels file with all othe
...
+                              [-topk {1,2,3,4,5,6,7,8,9,10}]
+                              [-nireq NUM_INFER_REQUESTS]ass
```
Comment (anzhella-pankratova) [inline demos/classification_demo/python/README.md:136]:
```suggestion
                              [-nireq NUM_INFER_REQUESTS]
```

File: demos/classification_demo/python/README.md [hunk 5]
```diff
@@ -130,60 +130,82 @@ and `<omz_dir>/data/dataset_classes/imagenet_2012.txt` labels file with all othe
 Running the application with the `-h` option yields the following usage message:
 
 ```
-usage: classification_demo.py [-h] -m MODEL -i INPUT [-d DEVICE] --labels LABELS [-ntop {1,2,3,4,5,6,7,8,9,10}]
-                              [-nireq NUM_INFER_REQUESTS] [-nstreams NUM_STREAMS] [-nthreads NUM_THREADS] [--loop]
-                              [-o OUTPUT] [-limit OUTPUT_LIMIT] [--no_show] [--output_resolution OUTPUT_RESOLUTION]
-                              [-u UTILIZATION_MONITORS] [--reverse_input_channels]
+usage: classification_demo.py [-h] -m MODEL [--adapter {openvino,remote}] -i
+                              INPUT [-d DEVICE] [--labels LABELS]
```
Comment (ivikhrev) [inline demos/classification_demo/python/README.md:134]:
```suggestion
usage: classification_demo.py [-h] -m MODEL [--adapter {openvino,remote}]
                               -i INPUT [-d DEVICE] [--labels LABELS]
```

**Comment by ivikhrev** [comment]
Jenkins pleas retry a build

**Comment by ivikhrev** [comment]
Jenkins please retry a build

File: demos/classification_demo/python/README.md (hunk 1 above)
Comment (Wovchena) [reply demos/classification_demo/python/README.md]: You should update https://github.com/openvinotoolkit/open_model_zoo/blob/e71b236ca0cdaa03b09f1352e61aa04e8c41188d/demos/classification_demo/python/classification_demo.py#L95 to have `.0` in first place, and regenerate README after that.

**Review by anzhella-pankratova — APPROVED** [review]

File: demos/classification_demo/python/README.md (hunk 1 above)
Comment (ivikhrev) [reply demos/classification_demo/python/README.md]: Done.

**Comment by ivikhrev** [comment]
Closed, reopen with branch from fork #2962

**Pull request closed without merging**